packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -144,8 +144,9 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
           [styles.compositeLinkIsExpanded]: link.isExpanded,
           [styles.compositeLinkIsSelected]: isLinkSelected
         }) }>
-        { (nestingLevel === 0 && link.links && link.links.length > 0 ?
```

Comment:
you need to include a rush change file; run `npm run change` which should be enough if your origin/master is updated.